packages/template-retail-react-app/app/pages/cart/index.jsx

```diff
@@ -86,6 +86,80 @@ const Cart = () => {
 
     const {data: customer} = useCurrentCustomer()
     const {customerId, isRegistered} = customer
+
+    /***************** Product Bundles ************************/
```

Comment:
Are you planning to add tests for the Cart page too? So for example, we can cover this scenario outlined in the PR description:
- Go to cart page with View Cart button
- increase quantity to 4 in product line item for product bundle on cart page
- Observe Only 3 left for Swing Tank! message appears

Reply:
Test has been added that goes over this case